crates/orderbook/openapi.yml

```diff
@@ -687,7 +687,7 @@ components:
         appData:
           $ref: "#/components/schemas/AppDataHash"
         feeAmount:
-          description: feeRatio * sellAmount + minimal_fee in atoms.
+          description: sellAmount in atoms to cover network fees. Needs to be zero (and incorporated into the limit price) when placing the order
```

Comment:
Thanks!

I brought also this becasue I was searching for the documentation of the `feeAmount` returned by the `quote` endpoint, but I shared this one that is not it.

I'm afraid we re-use the type for the order creation and the quote, and the fee logic is different. 
In the quote this field is an estimation given the current market conditions, in the creation should be set to zero as you mention. 

<img width="1091" alt="image" src="https://github.com/user-attachments/assets/5db7b04b-9a28-4ab4-94b1-7d17622da341" />